test/unit/events/notificationHandler.spec.ts

```diff
@@ -208,4 +208,33 @@ describe('Event Handler Tests', () => {
     assert.isUndefined(handledEvent);
     assert.equal(notificationCount, postEventNotificationCount);
   });
+
+  it('should not emit notification for excluded events', async () => {
+    // setup
+    const event: CWEvent = {
+      blockNumber: 10,
+      data: {
+        kind: SubstrateTypes.EventKind.DemocracyStarted,
```

Comment:
Are DemocracyStarted and DemocracyTabled different events?

Reply:
Yes but this is just an example test case. Note that I pass "DemocracyStarted" as the ignored event in the handler.